Common/tests/TestBase/SessionTableTestBase.cs

```diff
@@ -454,4 +443,38 @@ public async Task ListSessionAsyncNoFilterShouldSucceed()
                       res.Count);
     }
   }
+
+
+  [Test]
+  public async Task ListSessionAsyncTaskOptionsOptions()
```

Comment:
This tests the number of sessions, not that the sort was done correctly

Reply:
Yes, that is the role of the database.

Reply:
It mainly tests that the request is valid